src/features/cartographie/components/lieux-mediation-numerique-details/accessibilite/accessibilite.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="bg-light p-2 d-print-none d-flex">
```

Comment:
```suggestion
<div class="bg-light p-3 d-print-none d-flex">
```
J'ai l'impression que la marge interne est un peu fine par rapport au proto